pyccel/ast/numpyext.py

```diff
@@ -1160,8 +1160,8 @@ def is_elemental(self):
     'complex64' : NumpyComplex64,
     'matmul'    : NumpyMatmul,
     'sum'       : NumpySum,
-    'max'       : NumpyMax,
-    'min'       : NumpyMin,
+    'amax'       : NumpyAmax,
+    'amin'       : NumpyAmin,
```

Comment:
`np.min` seems to exist as an alias for `np.amin`. Try `np.min?` in ipython to see this. We should be able to keep both and the clearer class name `NumpyAmin` by doing:
```
'min'   : NumpyMin,
'amin'   : NumpyMin,
```

Reply:
Sorry, my typo has led to confusion. I meant:
``` 
'min'   : NumpyAmin,
'amin'  : NumpyAmin,
```
that way we keep the clearer names that you gave